src/stacks-hierarchy/Root_PurchaseOverviewScreen/Root_PurchaseOverviewScreen.tsx

```diff
@@ -127,17 +128,24 @@ export const Root_PurchaseOverviewScreen: React.FC<Props> = ({
   const focusRefs = useFocusRefs(params.onFocusElement);
 
   return (
-    <View style={styles.container}>
-      <FullScreenHeader
-        title={getTextForLanguage(params.fareProductTypeConfig.name, language)}
-        leftButton={{
+    <FullScreenView
+      headerProps={{
+        title: getTextForLanguage(params.fareProductTypeConfig.name, language),
+        leftButton: {
           type: 'cancel',
           onPress: closeModal,
-        }}
-        globalMessageContext={GlobalMessageContextEnum.appTicketing}
-        setFocusOnLoad={!params.onFocusElement}
-      />
-
+        },
+        setFocusOnLoad: !params.onFocusElement,
+        globalMessageContext: GlobalMessageContextEnum.appTicketing,
+      }}
+      parallaxContent={(focusRef?: React.MutableRefObject<null>) => (
+        <FareProductHeader
+          ref={focusRef}
```

Comment:
Maybe not set `foucsRef` if there is an `onFocusElement`? 🤔 You can test when going back from the zone selector.

Reply:
It works as expected, although I have to admit I haven't fully gotten the hang of this accessibility stuff. Whether that `focusRef` should actually be there, I honestly don't know. 

When testing this I've found that navigating with voice over in general has some strange issues and inconsistencies. I believe we (@HanneLH ?) created a task for going through accessibility and focus in general, but I can't find this at the moment. I recommend that a review of this should be prioritised.

Reply:
@KristianSelnas  We have two different task for screen reader.
https://github.com/AtB-AS/kundevendt/issues/14120 - For scooter parking.
https://github.com/AtB-AS/kundevendt/issues/10921 -  Collective task for screen reader issue

Reply:
I see there are now three competing focuses:
- The original screen header
- The parallax content
- The on focus element set when navigating.

We should clean this up a little. Just reach out if we should talk it through.